what4/src/What4/Expr/Builder.hs

```diff
@@ -384,6 +386,69 @@ type instance SymExpr (ExprBuilder t st fs) = Expr t
 type instance BoundVar (ExprBuilder t st fs) = ExprBoundVar t
 type instance SymAnnotation (ExprBuilder t st fs) = Nonce t
 
+exprCounter :: Getter (ExprBuilder t st fs) (NonceGenerator IO t)
```

Comment:
The type of `exprCounter` has changed from an ordinary field selector to a `Getter`, which is worth mentioning in the changelog. Similarly for `cacheStartSize` and `userState`.